### PR TITLE
Feature/core 1129 manage identity tests

### PR DESCRIPTION
**Pull request by Tschuck**
-  added basic identity management tests
- added basic identity switch tests
- fixed queue db by switching to identity
- added missing id's to ui

### CODE REVIEW

File: core/evancore.vue.libs/src/components/forms/checkbox/checkbox.vue [hunk 1]
```diff
@@ -38,7 +38,7 @@
         type="checkbox"
         v-bind="$attrs"
       >
-      <label :for="id" />
+      <label />
```
Comment (datrinh) [inline core/evancore.vue.libs/src/components/forms/checkbox/checkbox.vue:41]: Is this really sufficient? Will a click on the label focus the corresponding input?

File: core/ui.libs/src/utils/Queue.ts [hunk 2]
```diff
@@ -61,6 +61,11 @@ export default class EvanQueue {
    * @param      {number}  version  current db version
    */
   async openDB(version?: number) {
+    if (queueDB) {
+      queueDB.close();
+      await new Promise((resolve) => setTimeout(resolve, 50));
```
Comment (datrinh) [inline core/ui.libs/src/utils/Queue.ts:66]: What's the 50 ms timeout for? :)

File: tests-e2e/step-definitions/core/core.js [hunk 3]
```diff
@@ -48,6 +48,20 @@ Then(/^I want to see an element with class "([^"]+)"$/, async (className) => {
...
 });
 
+Then(/^I want to see an element with class "([^"]+)" and text "([^"]+)"$/, async (className, text) => {
+  client.useXpath();
+  testUtils.setupEvan(client);
+
+  await client.waitForElementPresent(`//div[@class="${className}"]/*[contains(., "${text}")]`, WAIT_TIME);
+});
+
+Then(/^I not want to see an element with class "([^"]+)" and text "([^"]+)"$/, async (className, text) => {
```
Comment (datrinh) [inline tests-e2e/step-definitions/core/core.js:58]:
```suggestion
Then(/^I do not want to see an element with class "([^"]+)" and text "([^"]+)"$/, async (className, text) => {
```

File: tests-e2e/step-definitions/core/vue.js [hunk 4]
```diff
@@ -67,20 +67,17 @@ Given('I go to the evan.network startpage', async () => {
...
-    await client.url(`${evan.baseUrl}#/dashboard.vue.evan`);
-    await client.waitForElementPresent('#evan-dapp-profile', 10 * 1000);
-    await client.click('#evan-dapp-profile');
-    await client.waitForElementPresent('#evan-logout', 10 * 1000);
-    await client.click('#evan-logout');
-    await client.waitForElementPresent('.modal-content #submit-logout', 10 * 1000);
-    await client.click('#submit-logout');
-  }
+  const evan = testUtils.setupEvan(client);
+  const language = global.tags && global.tags.includes('@german') ? 'de' : 'en';
```
Comment (datrinh) [inline tests-e2e/step-definitions/core/vue.js:73]: Prefer an "if" here. This reads like `language` could become a boolean, while the name suggests it being string.

File: tests-e2e/step-definitions/standard-ui/form.js [hunk 5]
```diff
@@ -143,7 +143,7 @@ When('I click on vue checkbox control with id {string}',
   async (id) => {
     client.useXpath();
 
-    const selector = `//*[@id="${id}"]/following-sibling::label[@for="${id}"]`;
```
Comment (datrinh) [inline tests-e2e/step-definitions/standard-ui/form.js:146]: The label always having a `for` attribute would be a good thing. Would be doing it another way be too much trouble?

File: core/evancore.vue.libs/src/components/forms/checkbox/checkbox.vue (hunk 1 above)
Comment (Tschuck) [reply core/evancore.vue.libs/src/components/forms/checkbox/checkbox.vue:41]: The label at this point is just for styling the "fancy" checkbox. When the id property was specified, the design does not change anymore, so it's not needed for the label.

File: core/ui.libs/src/utils/Queue.ts (hunk 2 above)
Comment (Tschuck) [reply core/ui.libs/src/utils/Queue.ts:66]: When we directly want to open the db again, we will get the indexeddb blocked error. An a onclose event isn't fired, when we are closing the connection by ourself. As a workaround, added the timeout and reopen works ;)

File: tests-e2e/step-definitions/standard-ui/form.js (hunk 5 above)
Comment (Tschuck) [reply tests-e2e/step-definitions/standard-ui/form.js:146]: Described in the first comment.

File: tests-e2e/step-definitions/core/vue.js (hunk 4 above)
Comment (Tschuck) [reply tests-e2e/step-definitions/core/vue.js:73]: Was just copied from before. I can change it

Comment (datrinh) [reply tests-e2e/step-definitions/core/vue.js]: Ah OK :)

**Pull request merged**